mathisfun/Complex.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+class Complex {
```

Comment:
Document the class.
Also, do you want to allow people to inherit from your class?
If not - use the 'final' specifier.